decidim-assemblies/app/views/layouts/decidim/admin/_manage_assemblies.html.erb

```diff
@@ -0,0 +1,44 @@
+<div class="inline-block relative">
+  <%= button_tag id: "assemblies-menu-trigger", data: { component: "dropdown", target: "assemblies-dropdown-menu-settings" }, class: "dropdown__trigger button button__transparent" do %>
+        <span>
+          <%= t("menu.manage", scope: "decidim.admin") %>
+        </span>
+    <%= icon "arrow-down-s-line" %>
+  <% end %>
```

Comment:
Seems like the indentation is off here
```suggestion
    <span>
      <%= t("menu.manage", scope: "decidim.admin") %>
    </span>
    <%= icon "arrow-down-s-line" %>
  <% end %>
```

Hopefully with #11707 this would be detected automatically 🚀